index.js

```diff
@@ -77,22 +81,56 @@ var tss;
             return path.dirname(require.resolve('typescript'));
         };
         TypeScriptSimple.prototype.getDefaultLibFilename = function (options) {
-            if (options.target === 2 /* ES6 */) {
+            if (options.target === ts.ScriptTarget.ES6) {
```

Comment:
@basarat I think this is better, but my tsc (TypeScript 1.4.2 on Mac) compiles `ts.ScriptTarget.ES6` to `2` without asking.
What your compiler option keeps enum expressions?